Autopilot/Inc/IMU.hpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * IMU Sensor Functions and Part Number Selection
+ * 
+ */
+
+//#pragma once
```

Comment:
We should either go with #pragma once or the #ifndef stuff. No need to keep the other commented.